src/test/kotlin/com/exactpro/epfast/template/dsl/FastSequenceBuilder.kt

```diff
@@ -18,7 +18,7 @@ package com.exactpro.epfast.template.dsl
 
 import com.exactpro.epfast.template.simple.Sequence
 
-class FastSequenceBuilder internal constructor(name: String, namespace: String) :
+class FastSequenceBuilder internal constructor(name: String?, namespace: String) :
```

Comment:
If this change (`String` -> `String?` isn't required by your code, roll back it). Name should be mandatory in all cases but sequence length. Ana works on fixing it now.